README.md

```diff
@@ -55,6 +58,43 @@ Options:
 
 - `kubectl`: Required for interacting with Kubernetes clusters.
 
+### IRSA Configuration Check (`/checks/kube/aws-irsa.sh`)
+
+##### Description:
+
+This script checks the IRSA (IAM Roles for Service Accounts) configuration for AWS Kubernetes. It ensures that the necessary components are configured correctly for OpenSearch and PostgreSQL.
+
+##### Usage:
+```bash
+Usage: ./checks/kube/aws-irsa.sh [-h] [-n NAMESPACE] [-e EXCLUDE_COMPONENTS] [-p COMPONENTS_PG] [-l COMPONENTS_OS] [-s]
+Options:
+  -h                              Display this help message
+  -n NAMESPACE                    Specify the namespace to use
+  -e EXCLUDE_COMPONENTS           Comma-separated list of components to exclude from the check (reference of the component is the root key used in the chart)
+  -p COMPONENTS_PG                Comma-separated list of components to check IRSA for PostgreSQL (overrides default list)
+  -l COMPONENTS_OS                Comma-separated list of components to check IRSA for OpenSearch (overrides default list)
+  -s                              Disable pod spawn for IRSA and network flow verification
+```
+
+##### Example:
+```bash
+./checks/kube/aws-irsa.sh -n camunda-primary -p "identity,webModeler" -l "zeebe,operate"
```

Comment:
curious how precise does the naming have to be?
Woudln't it be easier to just convert everything to lowercase, so customers can supply it however they want and we also don't have to care too much about special cases à la `webModeler`.

Maybe easier for the helm chart handling, just curious.

Reply:
was just an idea, I'm also fine with keeping it as is.